.travis.yml

```diff
@@ -15,27 +19,35 @@ matrix:
         - php: 5.5
           env: SYMFONY_VERSION=2.8.*
         - php: 5.6
-          env: SYMFONY_VERSION=3.0.*
-        - php: 7.0
           env: SYMFONY_VERSION=3.1.*
         - php: 7.1
-          env: SYMFONY_VERSION=3.2.*
-        - php: hhvm
+          env: PHP_CS_FIXER=true
+        - php: 7.1
```

Comment:
How about adding prefer-lowest here as well?

Reply:
prefer-lowest fails because of phpunit 4, I'll added in a next PR

Reply:
If the problem is only the TestCase class you can set the minimum phpunit to version 4.8.35
https://github.com/sebastianbergmann/phpunit/blob/master/ChangeLog-4.8.md